apps/web/lib/signup/getServerSideProps.tsx

```diff
@@ -17,9 +17,10 @@ const checkValidEmail = (email: string) => z.string().email().safeParse(email).s
 const querySchema = z.object({
   username: z
     .string()
+    .min(1)
     .optional()
     .transform((val) => val || ""),
-  email: z.string().email().optional(),
+  email: z.string().email().min(1).optional(),
```

Comment:
no we don't want to add min(1) here. the issue is about improving the error message

Reply:
something like "Invalid Email"

Reply:
Got it. Yeah, I improved the messages in the schema.